client/reader/stream/featured-asset.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+export default class FeaturedAsset extends React.Component {
```

Comment:
yeah, i think it helps give us defense in depth against images being broken in weird ways.